src/Commands/ExportPostmanCommand.php

```diff
@@ -190,6 +196,18 @@ public function handle(): void
         $this->info('Postman Collection Exported: '.storage_path('app/'.$exportName));
     }
 
+    protected function handleCallable($callable)
+    {
+        try {
+            return $callable();
+        } catch (\Throwable $th) {
+            if ($this->stopOnErrors) {
+                throw $th;
+            }
+            \Illuminate\Support\Facades\Log::error($th->getMessage()."\n[stacktrace]\n".$th->getTraceAsString()."\n");
```

Comment:
Could we import this rather than inline please?

Reply:
Done in commit 0f01fc7